### PR TITLE
Adding repeat tests and java 8 support

### PR DESCRIPTION
**Pull request by tonyreigns**
- [x] I have considered the risk of behavior change or other zero migration impact (https://github.com/OpenLiberty/open-liberty/wiki/Behavior-Changes).
- [x] If this PR fixes an Issue, the description includes "Fixes #FILLMEIN" or "Resolves #FILLMEIN" (verify `release bug` label if applicable: https://github.com/OpenLiberty/open-liberty/wiki/Open-Liberty-Conventions).
- [x] If this PR resolves an external Known Issue (including APARS), the description includes "Fixes #FILLMEIN" or "Resolves #FILLMEIN".

- Supporting Java 8 for the telemetry logging FAT buckets. 
- Adding TelemetryRepeats for both buckets
- Minor cleanup
Related to #27711

### CODE REVIEW

**Comment by tonyreigns** [comment]
#build

**Comment by tonyreigns** [comment]
#build

File: dev/io.openliberty.microprofile.telemetry.2.0.logging.internal_fat/fat/src/io/openliberty/microprofile/telemetry/logging/internal_fat/TelemetryFFDCCheckpointTest.java [hunk 1]
```diff
@@ -52,6 +54,7 @@ public static void initialSetup() throws Exception {
      */
     @Test
     @ExpectedFFDC({ "java.lang.ArithmeticException" })
+    @SkipForRepeat({ TelemetryActions.MP14_MPTEL20_ID })
```
Comment (pgunapal) [inline dev/io.openliberty.microprofile.telemetry.2.0.logging.internal_fat/fat/src/io/openliberty/microprofile/telemetry/logging/internal_fat/TelemetryFFDCCheckpointTest.java]: probably good to add a comment why we are skipping.

File: dev/io.openliberty.microprofile.telemetry.2.0.logging.internal_fat/fat/src/io/openliberty/microprofile/telemetry/logging/internal_fat/TelemetryMessagesCheckpointTest.java [hunk 2]
```diff
@@ -42,6 +44,7 @@ public static void initialSetup() throws Exception {
      * Ensures Liberty messages are correctly bridged and all attributes are present.
      */
     @Test
+    @SkipForRepeat({ TelemetryActions.MP14_MPTEL20_ID })
```
Comment (pgunapal) [inline dev/io.openliberty.microprofile.telemetry.2.0.logging.internal_fat/fat/src/io/openliberty/microprofile/telemetry/logging/internal_fat/TelemetryMessagesCheckpointTest.java]: Add a comment on why we are skipping.

File: dev/io.openliberty.microprofile.telemetry.2.0.logging.internal_fat/fat/src/io/openliberty/microprofile/telemetry/logging/internal_fat/TelemetryTraceCheckpointTest.java [hunk 3]
```diff
@@ -46,6 +48,7 @@ public static void initialSetup() throws Exception {
      * Ensures trace logs are bridged and all attributes are present.
      */
     @Test
+    @SkipForRepeat({ TelemetryActions.MP14_MPTEL20_ID })
```
Comment (pgunapal) [inline dev/io.openliberty.microprofile.telemetry.2.0.logging.internal_fat/fat/src/io/openliberty/microprofile/telemetry/logging/internal_fat/TelemetryTraceCheckpointTest.java]: Add a comment why we are skipping.

**Review by pgunapal — CHANGES_REQUESTED** [review]
Minor comments

**Review by pgunapal — DISMISSED** [review]
LGTM

**Pull request closed without merging**